assoc-aggregate/assoc-aggregate.wdl

```diff
@@ -517,90 +516,71 @@ task sbg_prepare_segments_1 {
 			return chrom_num
 
 		def pair_chromosome_gds(file_array):
+			# Corresponds with pair_chromosome_gds() in CWL
 			gdss = dict() # forced to use constructor due to WDL syntax issues
 			for i in range(0, len(file_array)): 
 				# Key is chr number, value is associated GDS file
-				this_chr = find_chromosome(file_array[i])
-				if this_chr == "X":
-					gdss[23] = os.path.basename(file_array[i])
-				elif this_chr == "Y":
-					gdss[24] = os.path.basename(file_array[i])
-				elif this_chr == "M":
-					gdss[25] = os.path.basename(file_array[i])
-				else:
-					gdss[int(this_chr)] = os.path.basename(file_array[i])
+				gdss[find_chromosome(file_array[i])] = os.path.basename(file_array[i])
+			logging.debug("pair_chromosome_gds returning %s" % gdss)
 			return gdss
 
 		def pair_chromosome_gds_special(file_array, agg_file):
+			# Corresponds with pair_chromosome_gds_special() in CWL
 			gdss = dict()
 			for i in range(0, len(file_array)):
-				gdss[int(find_chromosome(file_array[i]))] = os.path.basename(agg_file)
+				gdss[find_chromosome(file_array[i])] = os.path.basename(agg_file)
+			logging.debug("pair_chromosome_gds_special returning %s" % gdss)
 			return gdss
 
 		def wdl_get_segments():
 			segfile = open(IIsegments_fileII, 'rb')
 			segments = str((segfile.read(64000))).split('\n') # CWL x.contents only gets 64000 bytes
 			segfile.close()
-			segments = segments[1:] # remove first line
+			segments = segments[1:] # segments = segments.slice(1) in CWL; removes first line
 			return segments
 
-		######################
-		# prepare GDS output #
-		######################
-		beginning = datetime.datetime.now()
+		logging.debug("\n######################\n# prepare gds output #\n######################")
 		input_gdss = pair_chromosome_gds(IIinput_gds_filesII)
```

Comment:
`IIinput_gds_filesII`
Shudders in naming convention

Reply:
I hate it too. I want to make it very clear that it doesn't quite match the CWL version. What do you recommend using instead?

Reply:
Maybe `input_gds_files_py`?